coilsnake/assets/modulelist.txt

```diff
@@ -21,6 +21,7 @@ eb.WindowGraphicsModule
 eb.BattleBgModule
 eb.CompressedGraphicsModule
 eb.EnemyModule
+eb.DeathScreenModule
```

Comment:
imo this should go above SoundStoneModule. Since this likely compiles/decompiles very quickly it shoulg go earlier so it can fail fast